### PR TITLE
Define a matrix

### PR DESCRIPTION
**Pull request by StevenClontz**
For #382 

image

### CODE REVIEW

**Comment by StevenClontz** [comment]
@jkostiuk let me know what you think here. If you're not able to approve and squash/merge, then we should give you permissions to do so.

File: source/01-LE/01.ptx [hunk 1]
```diff
@@ -124,6 +124,69 @@ Following are some examples of addition and scalar multiplication in <m>\mathbb
     <definition>
         <statement>
             <p>
```
[the remark this answers is not in the document]
Comment (jkostiuk) [reply source/01-LE/01.ptx:126]: sorry for the lateness--i actually wrote this two weeks ago and it was "pending" :(

[the remark this answers is not in the document]
Comment (StevenClontz) [reply source/01-LE/01.ptx:126]: This brings up our inconsisentcy in at least one other place:

[Fact 3.2.10](https://teambasedinquirylearning.github.io/linear-algebra/2024e/AT2.html#AT3-standard-matrix-fact) 
uses $n\times m$ ($T:\mathbb R^m\to\mathbb R^n$).

I think I'm inclined to do this, but it leads to the conclusion that our linear systems should all have $m$ variables and $n$ equations, so it requires an audit of the book. I'll add that to this PR.

**Comment by StevenClontz** [comment]
image

File: source/01-LE/01.ptx (hunk 1 above)
[the remark this answers is not in the document]
Comment (jkostiuk) [reply source/01-LE/01.ptx:126]: yup, exactly!

**Review by jkostiuk — APPROVED** [review]

**Comment by StevenClontz** [comment]
Feel free to click the green "Squash and Merge" button next time you approve a PR (if you don't have the privilege to do so, we should fix it so you can).

**Comment by jkostiuk** [comment]
sounds good--I don't think I have permission FYI

**Comment by StevenClontz** [comment]
Nope, you didn't, but you should be able to now.

**Pull request merged**